### DISCUSSION TITLE
심사반려 => [솔루션관리] 페이지에서 JWT토큰 수정해주시기 바랍니다.  관련하여 문의 드립니다.

### DISCUSSION THREAD

**Discussion by billyLee1004** [commerce-api-naver/commerce-solutions#46] [묻고 답하기]
안녕하세요 셀롸이트 개발자 billy 입니다.

심사 중 아래 사항으로 반려가 되었습니다. 반려사항 중에 
1.[ 개발정보의 JWT 포함 링크가 정상적으로 열리지 않습니다. JWT 포함 링크는 개발사 자체 페이지로 이동되어야 합니다. 
[솔루션관리] 페이지에서 JWT토큰 수정해주시기 바랍니다. ]

=> 솔루션 관리 페이지에서 JWT토큰 수정을 어디서 하는 지 모르겠습니다. (* 저희는 승인형 솔루션 서비스 아닙니다. )
      솔루션 등록 페이지 말씀일까요? 수정은 없고 복사만 있습니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

모든 커머스솔루션은 구독자가 커머스솔루션마켓 또는 스마트스토어센터 솔루션관리 페이지에서 [사용하기]를 눌러 솔루션 웹 사이트로 이동할 때 자동으로 JWT 토큰이 전달됩니다.
이때 JWT 토큰 값은 URL 파라미터 `token` 파라미터로 전달되며, `token` 파라미터 값은 동적 값을 가집니다.

솔루션 심사 요청 시 `token` 파라미터가 포함된 내용으로 심사를 요청하셨을 경우 심사 결과에 영향을 미칠 수 있습니다.
보다 상세한 내용은 안내 받으신 내용에 포함된 [1:1 문의하기] 링크로 구체적인 수정 필요 항목과 방향을 확인해보시기 바랍니다.

> [!NOTE]
> 이 공간에서는 심사와 관련된 내용 외 기술적인 문의에 대해서만 확인이 가능합니다.

**Reply by billyLee1004** [reply]
빠른 답변 감사합니다.

솔루션 심사 요청 시 token 파라미터가 포함 하지 않았습니다. 
솔루션 사용연결 URL은 예를 들어 ["https://test.selltest.com/"] 으로 심사 요청했습니다. 

그리고 [1:1 문의 하기] 링크가 없는데 어디에 있습니까? 
![image](https://github.com/user-attachments/assets/d6d6ad2f-23e0-4506-a3fa-4c3302d4beea)

**Reply by commerce-api-naver** [reply]
@billyLee1004 님, 커머스API 이민석입니다.

먼저 솔루션 심사 결과 내 [1:1 문의하기] 링크는 조만간 제공 예정으로 정정드립니다.

말씀하신 내용으로 볼 때 심사 내용과 관련한 정확한 확인이 필요할 것으로 보입니다.
심사 시 제출하는 모든 정보는 실제 사용자 분들께 서비스됨을 가정하여 제출해주셔야 합니다.
예를 들어, 문의 본문에 말씀해주신 내용과 같이 외부 사용자가 접근 불가능한 URL(`https://test.selltest.com`)을 솔루션 정보로 포함하여 심사 요청을 주실 경우 심사 승인이 어려울 수 있습니다.

이 공간은 연동과 관련한 기술문의만 확인이 가능하기에 심사 결과에 대한 확인을 도와드리기 어렵습니다.
"[스마트스토어 고객센터 - 1:1문의하기](https://help.sell.smartstore.naver.com/mail/form.help?alias=sssolution)"로 전달받으신 심사 번호를 포함하여 궁금하신 점을 상세히 남겨주시면 정확한 확인을 도와드리도록 하겠습니다.
